tests/test_cases/test_force_release/test_force_release.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+import cocotb
+from cocotb.triggers import Timer
+from cocotb.result import TestFailure
+from cocotb.handle import Force, Release
+
+
+@cocotb.test(expect_fail=cocotb.SIM_NAME in ["GHDL"])
+def test_force_release(dut):
+    """
+    Test force and release on simulation handles
+    """
+    log = logging.getLogger("cocotb.test")
+    yield Timer(10)
```

Comment:
```suggestion
    yield Timer(10, "ns")
```
(and a few times more below). We try to establish this as best practice, we just haven't gotten around to changing the existing tests yet.

Reply:
Actually, you may even want to use https://cocotb.readthedocs.io/en/latest/coroutines.html#async-functions here to be state-of-the-art :)

Reply:
@thasti Did you see my second comment here? (Maybe I shouldn't have appended it.)

Reply:
Yes, was seen. Applied now as well!